unlock-app/src/components/interface/verifiers/VerifiersList.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { HiOutlineTrash as TrashIcon } from 'react-icons/hi'
+import { Button, Modal, Tooltip } from '@unlock-protocol/ui'
+import AuthenticationContext from '../../../contexts/AuthenticationContext'
+import { getAddressForName } from '../../../hooks/useEns'
+import { ToastHelper } from '../../helpers/toast.helper'
+import Loading from '../Loading'
+import { ConfigContext } from '../../../utils/withConfig'
+
+const styling = {
+  sectionWrapper: 'text-left mx-2 my-3',
+  sectionTitle: 'text-lg text-black font-bold',
+  sectionDesctiption: 'text-sm text-black-600 text-align-text',
+  address: 'text-indigo-700',
+  actions: 'flex mt-5 justify-center',
+  button: 'ml-2 flex',
+}
+interface VerifiersListProsps {
+  lockAddress: string
+}
+export const VerifiersList: React.FC<VerifiersListProsps> = ({
+  lockAddress,
+}) => {
+  const { network } = useContext(AuthenticationContext)
+  const [selectedVerifier, setSelectedVerifier] = useState('')
+  const [showDeleteVerifierModal, setShowDeleteVerifierModal] = useState(false)
+  const [loading, setLoading] = useState(false)
+  const [verifiers, setVerifiers] = useState<any[]>([])
+  const config: any = useContext(ConfigContext)
+
+  const setDefaults = () => {
+    setShowDeleteVerifierModal(false)
+    setSelectedVerifier('')
+    setLoading(false)
+  }
+
+  useEffect(() => {
+    getVerifierList()
+  }, [])
+
+  const getVerifierList = async () => {
+    try {
+      const addVerifierUrl = `${config.services.storage.host}/v2/api/verifier/${network}/${lockAddress}/${selectedVerifier}`
+      const requestOptions = {
+        method: 'GET',
+        headers: { 'Content-Type': 'application/json' },
+      }
+      await fetch(addVerifierUrl, requestOptions)
+        .then((res) => res.json())
+        .then((verifiers: any) => {
+          setVerifiers(verifiers)
+        })
+    } catch (err: any) {
+      setLoading(false)
+      ToastHelper.error(
+        err?.error ??
+          'There was a problem adding verifier, please re-load and try again'
+      )
+    }
+  }
+  const onConfirmDeleteVerifier = async () => {
+    setLoading(true)
+    const resolvedAddress = await getAddressForName(selectedVerifier)
```

Comment:
Can we use the address directly and not the ENS here? I assume we store addresses and not ENS on our end.

Reply:
ok! nice